x/mongo/driver/operation/hello.go

```diff
@@ -125,99 +124,6 @@ func (h *Hello) Result(addr address.Address) description.Server {
 	return description.NewServer(addr, bson.Raw(h.res))
 }
 
-const (
```

Comment:
Moved all of this logic to the driverutil internal package so it can be shared across multiple x packages without duplication.